Mathlib/Order/Concept.lean

```diff
@@ -113,19 +113,11 @@ theorem extentClosure_iUnion (f : ι → Set β) :
   intentClosure_iUnion _ _
 #align extent_closure_Union extentClosure_iUnion
 
-/- ./././Mathport/Syntax/Translate/Expr.lean:107:6: warning: expanding binder group (i j) -/
-/- ./././Mathport/Syntax/Translate/Expr.lean:107:6: warning: expanding binder group (i j) -/
```

Comment:
I think we still want to fix those once the syntax is supported, but perhaps we can find them without porting notes